source/install/install-rhel-66-mysql.rst

```diff
@@ -0,0 +1,66 @@
+..  _install-rhel-66-mysql:
+
+Installing MySQL Database Server
+================================
+
+Install and set up the database for use by the Mattermost server. You can install either MySQL or PostgreSQL. To install PostgreSQL, see :ref:`install-rhel-66-postgresql`.
```

Comment:
It's too bad that the Sphinx processor doesn't create the link properly. I think in this case though, we can get rid of the reference altogether. The reference made more sense in the modular structure, but in the monolith it's a bit redundant, especially in this particular case with the PostgreSQL section and the MySQL section right beside each other in the ToC and in the document.